modules/swagger-jaxrs/src/main/java/io/swagger/jaxrs/Reader.java

```diff
@@ -1001,9 +987,10 @@ private void processOperationDecorator(Operation operation, Method method) {
 
     private void addResponse(Operation operation, ApiResponse apiResponse) {
         Map<String, Property> responseHeaders = parseResponseHeaders(apiResponse.responseHeaders());
+        Map<String, Object> examples = parseExamples(apiResponse.examples());
 
         Response response = new Response()
-        .description(apiResponse.message()).headers(responseHeaders);
+        .description(apiResponse.message()).headers(responseHeaders).setExamples(examples);
```

Comment:
would call `response.setExamples()` on new line, with no need of modifying setter behaviour in response POJO

Reply:
Sure, I will change that.